### PR TITLE
feat(ci): use github actions

### PR DESCRIPTION
**Pull request by jeremad**

### CODE REVIEW

File: .github/workflows/checks.yml [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+      - name: Install dmenv
+        uses: TankerHQ/dmenv-install-action@v1.0.0
+        with:
+          dmenv-version: 0.20.0
+      - name: Prepare project for development
+        run: dmenv install
+      - name: Run tests
+        run: dmenv run -- pytest
+
+  python34:
```
Comment (dmerejkowsky) [inline .github/workflows/checks.yml]: why is python3.4 special?

File: README.rst [hunk 2]
```diff
@@ -3,9 +3,6 @@ Identity SDK
 
 Tanker identity generation in Python for the `Tanker SDK <https://tanker.io/docs/latest>`_.
 
-.. image:: https://travis-ci.org/TankerHQ/identity-python.svg?branch=master
-    :target: https://travis-ci.org/TankerHQ/identity-python
-
```
Comment (dmerejkowsky) [inline README.rst]: you can add a github action badge next

File: .github/workflows/checks.yml (hunk 1 above)
Comment (jeremad) [reply .github/workflows/checks.yml]: It's not avaible on the latest ubuntu: https://help.github.com/en/actions/automating-your-workflow-with-github-actions/software-installed-on-github-hosted-runners#ubuntu-1804-lts

Comment (dmerejkowsky) [reply .github/workflows/checks.yml]: Fair enough

**Pull request merged**